lib/salus/scanners/yarn_audit.rb

```diff
@@ -251,6 +386,16 @@ def scan_deps
       command << 'optionalDependencies ' unless dep_types.include?('optionalDependencies')
     end
 
+    def find_nested_hash_value(obj, key)
```

Comment:
It might be safer to just check if `obj.is_a?(Hash)`? 
Maybe some other random class also responds to `key?`